src/node/rpc/frontend.h

```diff
@@ -73,12 +73,47 @@ namespace ccf
       handlers.set_history(history);
     }
 
+    std::vector<uint8_t> get_cert_to_forward(
```

Comment:
This is the main change in this PR. We no longer override this function in each frontend but base it on `handlers` and the `handler`'s `require_client_identity` flag.